[GeoMechanicsApplication] Fixed several (minor) issues detected by Clang-tidy

avdg81 opened the pull request:
**📝 Description**
Fixed several (minor) issues detected by Clang-tidy.

**🆕 Changelog**
- Make sure that parameter names are declared consistently.
- Made an unused function parameter "unnamed".
- Applied the Kratos Style Guide to a few parameter names.
- Removed some redundant size checks.
- Avoid repeating the return type from the declaration; use a braced initializer list instead.
- Formatted the C++ code using Clang-format.

avdg81 committed:
Fixed a few minor issues

- Avoid repeating the return type from the declaration; use a braced initializer list instead.
- Formatted the C++ code.

avdg81 committed:
Make sure that parameter names are declared consistently

Other changes include:
- Made an unused function parameter "unnamed".
- Applied the Kratos Style Guide to a few parameter names.
- Removed some redundant size checks.
- Formatted the code using Clang-format.

WPK4FEM commented on applications/GeoMechanicsApplication/custom_elements/U_Pw_base_element.cpp:
```diff
@@ -411,18 +411,18 @@ std::vector<double> UPwBaseElement::CalculateIntegrationCoefficients(
 
-    GeometryUtils::JacobianOnInitialConfiguration(rGeom, IntegrationPoints[IntegrationPointIndex], J0);
+    GeometryUtils::JacobianOnInitialConfiguration(rGeom, IntegrationPoints[IntegrationPointIndex], rJ0);
     const Matrix& DN_De = rGeom.ShapeFunctionsLocalGradients(mThisIntegrationMethod)[IntegrationPointIndex];
```
Should there be a ref in the name here too? ( You have corrected the header argument names ) Here I expex r_dn_de

avdg81 replied:
Yes, to comply with the Kratos Style Guide, we should. Changed.

WPK4FEM commented on applications/GeoMechanicsApplication/custom_elements/U_Pw_base_element.hpp:
```diff
@@ -166,8 +166,11 @@ class KRATOS_API(GEO_MECHANICS_APPLICATION) UPwBaseElement : public Element
+    void CalculateDerivativesOnInitialConfiguration(double&      rDetJ,
+                                                    Matrix&      rJ0,
+                                                    Matrix&      rInvJ0,
+                                                    Matrix&      rDNu_DX0,
```
Indeed the function name indicates that this should be 0.

WPK4FEM commented on applications/GeoMechanicsApplication/custom_elements/steady_state_Pw_piping_element.cpp:
```diff
@@ -283,25 +283,24 @@ double SteadyStatePwPipingElement<3, 8>::CalculateHeadGradient(const PropertiesT
+    double particle_d = GeoTransportEquationUtilities::CalculateParticleDiameter(rProperties);
 
     // todo calculate slope of pipe, currently pipe is assumed to be horizontal
     const double pipeSlope = 0;
```
Then probably all const auto i.s.o. const double, but that may be out of scope for this PR.

avdg81 replied:
Yes, I agree. In this case, we can even inline the variables, which saves a few intermediate variables without loosing clarity. That's what I have done. Let me know what you think.

WPK4FEM commented on applications/GeoMechanicsApplication/custom_elements/updated_lagrangian_U_Pw_diff_order_element.cpp:
```diff
@@ -25,16 +25,15 @@ Element::Pointer UpdatedLagrangianUPwDiffOrderElement::Create(IndexType
                                                               NodesArrayType const& ThisNodes,
```
ThisNodes -> rNodes?

avdg81 replied:
Done.

WPK4FEM commented on applications/GeoMechanicsApplication/custom_processes/apply_excavation_process.cpp:
```diff
@@ -22,10 +22,10 @@
 {
 
-ApplyExcavationProcess::ApplyExcavationProcess(ModelPart& rModelPart, const Parameters& rProcessSettings)
+ApplyExcavationProcess::ApplyExcavationProcess(ModelPart& rModelPart, const Parameters& rSettings)
```
maybe rProcessParameters is a more descriptive name than rSettings or rProcessSettings.

avdg81 replied:
Renamed to `rProcessSettings`.

WPK4FEM commented on applications/GeoMechanicsApplication/custom_processes/set_multiple_moving_loads.h:
```diff
@@ -48,7 +48,7 @@ class KRATOS_API(GEO_MECHANICS_APPLICATION) SetMultipleMovingLoadsProcess : publ
     ///@{
 
-    SetMultipleMovingLoadsProcess(ModelPart& rModelPart, const Parameters& rParameters);
+    SetMultipleMovingLoadsProcess(ModelPart& rModelPart, const Parameters& rSettings);
```
Maybe here too rProcessParameters

avdg81 replied:
Done.

WPK4FEM commented on applications/GeoMechanicsApplication/custom_strategies/builder_and_solvers/residualbased_block_builder_and_solver_with_mass_and_damping.h:
```diff
@@ -107,7 +107,7 @@ class ResidualBasedBlockBuilderAndSolverWithMassAndDamping
     using ConditionsArrayType   = typename BaseType::ConditionsArrayType;
 
     /// Additional definitions
```
why /// i.s.o. //

avdg81 replied:
I have no clue. Perhaps this reflects the personal preference of the authors? ;-)

WPK4FEM reviewed (commented):
This looks like a nice step towards more consistency. Some suggestions, but they may well be kept out of this scope.

avdg81 committed:
Processed Wijtze Pieter's review suggestions

- Renamed a few more variables to comply with the Kratos Style Guide. Also renamed a few function parameters.
- Inlined a few variables.

avdg81 committed:
Removed redundant comment

avdg81 committed:
Reverted changes

avdg81 committed:
Reapplied changes

avdg81 committed:
Update updated_lagrangian_U_Pw_diff_order_element.hpp

Removed two redundant comments

WPK4FEM reviewed (approved):
3x is scheepsrecht
